CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 Please add your changelog entry under this comment in the correct category (Security, Fixed, Added, Changed, Deprecated, Removed - in this order).
 -->
 
+### Changed
+* panic on 401 responses from engine to slow down requests with invalid token @marioreggiori (#155)
```

Comment:
please add the hint this only affects the instance controller for now, just prefixing "instances: " should be enough